packages/web/src/components/pages/WebsiteResults.tsx

```diff
@@ -87,9 +92,54 @@ export function WebsiteResultsPage() {
         dispatch(resetScanDisplayOptions({ scanUrl: normalizedUrl }));
       }
     },
-    [dispatch, normalizedUrl]
+    [dispatch, normalizedUrl, selectedDisplayOptions]
   );
 
+  const handleSortersChange = useCallback(
+    (newSorters: PackageSorter[]) => {
+      if (!normalizedUrl) {
+        return;
+      }
+
+      dispatch(
+        setScanDisplayOptions({
+          scanUrl: normalizedUrl,
+          options: {
+            ...selectedDisplayOptions,
+            packageSorters: newSorters,
+          },
+        })
+      );
+    },
+    [dispatch, normalizedUrl, selectedDisplayOptions]
+  );
+
+  const selectedSort = {
```

Comment:
This could be simplified as `const selectedSort = selectedDisplayOptions.packageSorters[0];`